server/common/transport-shared/src/main/java/org/kaaproject/kaa/server/transport/SpecificTransportContext.java

```diff
@@ -13,22 +13,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.kaaproject.kaa.server.operations.service.config;
+package org.kaaproject.kaa.server.transport;
 
-import org.kaaproject.kaa.common.bootstrap.gen.ChannelType;
+import org.apache.avro.specific.SpecificRecordBase;
 
 /**
- * @author Andrey Panasenko
+ * A context for transport initialization parameters with specific avro decoded configuration.
```

Comment:
Provides a context for the transport initialization parameters with the specific avro-decoded configuration.